rpgame/src/main/java/pk/rpgame/controller/ExplorationController.java

```diff
@@ -1,5 +1,83 @@
 package pk.rpgame.controller;
 
+import pk.rpgame.model.Room;
+import pk.rpgame.model.WorldEntity;
+import pk.rpgame.model.items.Armor;
+
+import pk.rpgame.model.items.ArmorItem;
+import pk.rpgame.model.items.Item;
+import pk.rpgame.model.items.Potion;
+import pk.rpgame.model.living.Hero;
+import pk.rpgame.view.*;
+
+import java.util.List;
+
 public class ExplorationController implements IController {
 
+
+    private static ExplorationView explorationView;
+    private Room room;
+
+    private Map map;
+
+    private GeneralMenu generalMenu;
+
+    private WorldEntity worldEntityModel;
+
+    private Hero heroControler;
+
+
+
+    public ExplorationController(Hero hero,WorldEntity worldEntity, Room currentRoom) {
+        this.explorationView = new ExplorationView();
```

Comment:
Jeśli explorationView jest static, więc można odpuścić ".this". Choć nie jestem pewna czy View powinno być statyczne

Reply:
Skoro nie jesteśmy pewni, to zostawiam